### PR TITLE
LANG-1443: Add more SystemUtils.IS_JAVA_XX variants

### PR DESCRIPTION
**Pull request by britter**

### CODE REVIEW

File: src/test/java/org/apache/commons/lang3/JavaVersionTest.java [hunk 1]
```diff
@@ -20,23 +20,10 @@
 
 import org.junit.jupiter.api.Test;
 
-import static org.apache.commons.lang3.JavaVersion.JAVA_RECENT;
+import static org.apache.commons.lang3.JavaVersion.*;
```
Comment (kinow) [inline src/test/java/org/apache/commons/lang3/JavaVersionTest.java]: Checkstyle ain't happy with this .* import I think.

Comment (garydgregory) [reply src/test/java/org/apache/commons/lang3/JavaVersionTest.java]: Right, we need specific imports.

Comment (chtompki) [reply src/test/java/org/apache/commons/lang3/JavaVersionTest.java]: IntelliJ does that by default at greater than 5 import statements. It rewrites them with the *

Comment (britter) [reply src/test/java/org/apache/commons/lang3/JavaVersionTest.java]: Damn you, IntelliJ! 😤Will fix that later today.

**Comment by britter** [comment]
Fixed the Checkstyle problems.

**Pull request merged**